cartography/intel/aws/route53.py

```diff
@@ -9,6 +9,7 @@ def link_aws_resources(neo4j_session, update_tag):
     # find records that point to other records
     link_records = """
     MATCH (n:AWSDNSRecord) WITH n MATCH (v:AWSDNSRecord{value: n.name})
+    WHERE NOT n = v
```

Comment:
Is this possible, or do you need to do something like `WHERE NOT n.id = v.id`?

Reply:
Yes, it is just checking that the node is not equal instead of just a property.